src/test/groovy/com/github/jengelman/gradle/plugins/shadow/ShadowPluginSpec.groovy

```diff
@@ -495,12 +495,12 @@ class ShadowPluginSpec extends PluginSpecification {
 
         file('lib/src/main/java/lib/LibEntity.java') << """
             package lib;
-            public interface LibEntity {}
+            public interface LibEntity { void simple(); }
```

Comment:
we should follow appropriate Java styling and put this on a new line for ease of reading. Same for all the changes in this file.